containerm/cli/cli_command_ctrs_list.go

```diff
@@ -69,6 +89,45 @@ func (cc *listCmd) setupFlags() {
 	flagSet := cc.cmd.Flags()
 	// init name flags
 	flagSet.StringVarP(&cc.config.name, "name", "n", "", "List all containers with a specific name.")
+	flagSet.BoolVarP(&cc.config.quiet, "quiet", "q", false, "List only container IDs.")
+	flagSet.StringSliceVar(&cc.config.filter, "filter", nil, "Lists only containers with a specified filter. The containers can be filtered by their status, image and exitcode.")
```

Comment:
Give example of a filtering.
```suggestion
	flagSet.StringSliceVar(&cc.config.filter, "filter", nil, "Lists only containers with a specified filter. The containers can be filtered by their status, image and exitcode. Example:\n--filter status=running")
```

Reply:
Sorry, there is already an example above.